index.rst

```diff
@@ -1,61 +1,106 @@
+However, if there is a separate transfer from the summit facility to the base facility, it may forgo encryption if that encryption could interfere with the requirements for Prompt Processing (in particular, the 60 second bound on elapsed time from shutter close to alert production).
+This is permissible because the connection from the summit facility to the base facility is over dedicated private fiber and the networking equipment at both ends is under the control of Vera C. Rubin Observatory.
+
+Communication from the :abbr:`DAQ (Data Acquisition System)` on the telescope to its client need not be (and is not) encrypted since this communication happens entirely within the summit facility.
+Only Rubin Observatory staff will have access to the summit facility or any of the systems running there.
+
+Project impact
+--------------
+
+Data transfer has previously been tested using bbcp_, a data transfer program for scientific data that is optimized for transfer speed.
+It encrypts the control channel (via SSH_) but not the data channel, and therefore does not meet this requirement.
+Data transfer will have to be reimplemented using TLS.
+
+.. _bbcp: https://www.slac.stanford.edu/~abh/bbcp/
+.. _SSH: https://en.wikipedia.org/wiki/Ssh_(Secure_Shell)
+
+To obtain sufficient transfer speeds over TLS, it is often necessary to chunk the data being transferred and transfer multiple chunks in parallel.
+See, for example `multipart upload for Amazon S3`_ or `parallel composite uploads with GCS`_, which offer advice for large file uploads over HTTP and TLS to users of Amazon's S3 and Google's Google Cloud Storage services.
+Cloud storage upload tools offer automatic facilities to chunk and upload large files in parallel.
+Rubin Observatory may need to implement similar functionality at the client and server ends of the transfer to achieve the necessary performance for data transfer to the USDF.
 
-.. TODO: Delete the note below before merging new content to the master branch.
+.. _multipart upload for Amazon S3: https://docs.aws.amazon.com/AmazonS3/latest/dev/uploadobjusingmpu.html
+.. _parallel composite uploads with GCS: https://cloud.google.com/storage/docs/gsutil/commands/cp
 
-.. note::
+Finally, encryption has some additional compute processing overhead in both the client and server.
+TLS uses AES encryption, which is highly optimized in modern processors, but it is still more expensive to encrypt and decrypt the data than to transfer it unencrypted.
+In order to meet the 60 second window for Prompt Processing, this may require more processing power at both ends of the transfer.
+However, this additional processing is expected to be a small fraction of the processing required to compress the images before transfer, a generally more expensive operation.
```

Comment:
We have tested with Google Cloud using the Google Cloud Storage API and achieved transfer times within the requirement (but only from a single machine as of now), including both compression and TLS, without using parallel composite uploads (but with retained HTTPS/TCP connections and disabled slow-start).  See DMTN-157.

Reply:
Thanks, I added this as an additional section.